<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Steganography</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../image/input/blocks.ppm</normaloff>../../image/input/blocks.ppm</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="HideData">
     <attribute name="title">
      <string>Hide Data</string>
     </attribute>
     <widget class="QLabel" name="EnterCoverImage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter cover image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Browse1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QLabel" name="EnterSecretFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter secret file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Browse2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewImage">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>View image</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt using AES-128</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="EnterKey">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DoIt">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Do It!!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>View stego image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>View Orignal image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RecoverData">
     <attribute name="title">
      <string>Recover Data</string>
     </attribute>
     <widget class="QLabel" name="EnterStegoImage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter stego image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Browse3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewImage2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>View image</string>
      </property>
     </widget>
     <widget class="QLabel" name="Enterkey2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>This data requires a key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="DoIt2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Do It!!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>EnterKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Browse1</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>GetCoverImageName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Browse2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>GetSecretFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewImage</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewCoverImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DoIt</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>Stegn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Browse3</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>GetStegoFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DoIt2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>Invstegn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewImage2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewStegoImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit1</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewPreviousFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewOriginalImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>GetCoverImageName()</slot>
  <slot>GetSecretFileName()</slot>
  <slot>ViewCoverImage()</slot>
  <slot>Stegn()</slot>
  <slot>GetStegoFileName()</slot>
  <slot>Invstegn()</slot>
  <slot>ViewStegoImage()</slot>
  <slot>ViewPreviousFile()</slot>
  <slot>ViewOriginalImage()</slot>
 </slots>
</ui>
